File: app/src/main/java/org/apache/fineract/data/models/product/ProductPage.java
package org.apache.fineract.data.models.product;

import java.util.List;

/**
 * @author Rajan Maurya
 *         On 20/07/17.
 */

public class ProductPage {

    private List<Product> elements;
    private Integer totalPages;
    private Long totalElements;

    public List<Product> getElements() {
        return elements;
    }

    public void setElements(List<Product> elements) {
        this.elements = elements;
    }

    public Integer getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(Integer totalPages) {
        this.totalPages = totalPages;
    }

    public Long getTotalElements() {
        return totalElements;
    }

    public void setTotalElements(Long totalElements) {
        this.totalElements = totalElements;
    }
}

File: app/src/main/java/org/apache/fineract/data/models/product/ProductPage.kt
package org.apache.fineract.data.models.product

import com.google.gson.annotations.SerializedName

/**
 * @author Rajan Maurya
 * On 20/07/17.
 */

data class ProductPage (
        @SerializedName("elements") val elements: List<Product>? = null,
        @SerializedName("totalPages") val totalPages: Int? = null,
        @SerializedName("totalElements") val totalElements: Long? = null
)
